src/components/Select/ComboboxMulti/ComboboxMulti.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/default-props-match-prop-types */
```

Comment:
Is it really necessary to have that esilnt-disable line here? Seems like everything is matched fine between defaultProps and propTypes. I just removed this line and the same from SelectBase, ran `npm run lint` and didn't get any warnings about them. Am I doing something wrong?